src/components/ComponentIndexPage/index.js

```diff
@@ -58,43 +59,94 @@ function sortByNewest(a, b) {
   return dateA - dateB;
 }
 
+const filterOptions = {
```

Comment:
You might be able to do transform this into an array like:

```js
const filterOptions = {
  [
    title: 'Framework',
    options: ['React', 'Angular', 'Vue', 'vanilla'],
  ],
  // ...
};
```

And then later on:

```jsx
        <fieldset className="bx--fieldset">
          {fieldOptions.map(({ title, options }) => (
            <div key={key} className="component-index-filter__option">
              <legend className="bx--label">{title}</legend>
              {options.map((option) => (
                <Checkbox
                  labelText={option}
                  id={option}
                  checked={selected.includes(option)}
                  onChange={(filterOption) =>
                    handleOnChange(filterOption, option)
                  }
                />
              ))}
            </div>
          ))}
        </fieldset>
```

To avoid that `Object.entries` bit and treat the data as more of a collection that you're iterating over